docs/usage.rst

```diff
@@ -26,6 +26,21 @@ Encoding & Decoding Tokens with RS256 (RSA)
     >>decoded = jwt.decode(encoded, public_key, algorithms='RS256')
     {'some': 'payload'}
 
+If your private key needs a passphrase, you need to pass in a ``PrivateKey`` object from ``cryptography``.
+
+.. code-block:: python
+
+    from cryptography.hazmat.primitives import serialization
+    from cryptography.hazmat.backends import default_backend
+
+    pem_bytes = b'-----BEGIN PRIVATE KEY-----\nMIGEAgEAMBAGByqGSM49AgEGBS...'
```

Comment:
is b'' needed?

Reply:
That line was copied from an already-existing line [here](https://github.com/jpadilla/pyjwt/pull/525/files#diff-c4b284a0e8c330264ebefb4d3d14dd5bb68fe3b72c2ae656f49132e2c134b84fR22). :-)

Reply:
I even dig into [cryptography's official doc](https://cryptography.io/en/latest/hazmat/primitives/asymmetric/serialization.html?highlight=load_pem_private_key#cryptography.hazmat.primitives.serialization.load_pem_private_key). They requires a bytes-like pem data. So, there is no problem of this line.

Can someone provide more reviews if any, or simply merge this in?